tvm/serialization/boc.mdx

```diff
@@ -2,6 +2,279 @@
 title: "Bag of cells"
 ---
 
-import { Stub } from '/snippets/stub.jsx';
+import { Aside } from '/snippets/aside.jsx';
 
-<Stub issue="229" />
+Arbitrary data are represented in the TON Blockchain by trees of cells. Such a tree of cells is transformed into a
+[DAG](https://en.wikipedia.org/wiki/Directed_acyclic_graph) of cells by identifying identical cells in the
+tree. After that, each of the references of each cell might be replaced by the
+32-byte [representation hash](/tvm/serialization/cells#standard-cell-representation-and-its-hash) of the cell referred to. Thus a _bag of cells (BoC)_ is obtained.
+In general, a BoC can be obtained from several trees of cells, thus forming a forest. By convention, the root of the original tree of
```

Comment:
if it's a forest, then why are we talking about "the root of the original tree"?